<?xml version="1.0" encoding="UTF-8" ?>
<ui version="4.0">
    <class>DisplayVideosWindow</class>
    <widget class="QWidget" name="DisplayVideosWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>560</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>DisplayVideosWindow</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(248, 249, 250);</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>390</x>
                    <y>60</y>
                    <width>601</width>
                    <height>371</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout" />
        </widget>
        <widget class="QListWidget" name="listWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>59</y>
                    <width>370</width>
                    <height>481</height>
                </rect>
            </property>
        </widget>
        <widget class="ResPushButton" name="btn_back">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>40</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
                <string />
            </property>
        </widget>
        <widget class="ResPushButton" name="btn_play">
            <property name="geometry">
                <rect>
                    <x>390</x>
                    <y>440</y>
                    <width>30</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
                <string />
            </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider">
            <property name="geometry">
                <rect>
                    <x>430</x>
                    <y>445</y>
                    <width>441</width>
                    <height>22</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    QSlider::groove:horizontal {
                    background:transparent;
                    }
                    QSlider::handle:horizontal {
                    background: rgb(255,255,255);
                    width:8px;
                    height:8px;
                    border-radius:4px;
                    margin-top:7px;
                    margin-bottom:7px;
                    }
                    QSlider::sub-page:horizontal {
                    background:rgb(255,255,255);
                    margin-top:9px;
                    margin-bottom:9px;
                    }
                    QSlider::add-page:horizontal {
                    background: rgb(56,56,56);
                    margin-top:9px;
                    margin-bottom:9px;
                    }
                    QSlider::handle:horizontal:pressed {
                    background: rgb(255,255,255);
                    width:8px;
                    height:8px;
                    border-radius:4px;
                    margin-top:7px;
                    margin-bottom:7px;
                    }
                </string>
            </property>
            <property name="pageStep">
                <number>1</number>
            </property>
            <property name="orientation">
                <enum>Qt::Horizontal</enum>
            </property>
        </widget>
        <widget class="ResPushButton" name="btn_modify">
            <property name="geometry">
                <rect>
                    <x>870</x>
                    <y>500</y>
                    <width>100</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
                <string />
            </property>
        </widget>
        <widget class="QLabel" name="label_positionTime">
            <property name="geometry">
                <rect>
                    <x>880</x>
                    <y>450</y>
                    <width>50</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string />
            </property>
        </widget>
        <widget class="QLabel" name="label_totalTime">
            <property name="geometry">
                <rect>
                    <x>940</x>
                    <y>450</y>
                    <width>50</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string />
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>930</x>
                    <y>450</y>
                    <width>8</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>/</string>
            </property>
        </widget>
        <widget class="QWidget" name="widget" native="true">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1000</width>
                    <height>560</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">background-color: rgb(248, 249, 250);</string>
            </property>
        </widget>
        <zorder>widget</zorder>
        <zorder>verticalLayoutWidget</zorder>
        <zorder>listWidget</zorder>
        <zorder>btn_back</zorder>
        <zorder>btn_play</zorder>
        <zorder>horizontalSlider</zorder>
        <zorder>btn_modify</zorder>
        <zorder>label_positionTime</zorder>
        <zorder>label_totalTime</zorder>
        <zorder>label</zorder>
    </widget>
    <layoutdefault spacing="6" margin="11" />
    <customwidgets>
        <customwidget>
            <class>ResPushButton</class>
            <extends>QPushButton</extends>
            <header location="global">F:\MIProject\MusicVisualization\MusicVisualization\src\Module\QtWidget\respushbutton.h</header>
        </customwidget>
    </customwidgets>
    <resources />
    <connections />
</ui>